tutorials/tts/FastPitch_Data_Preparation.ipynb

```diff
@@ -0,0 +1,1125 @@
+      ],
+      "metadata": {
+        "id": "x2yhJtsj2lDR"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "audio_file = \"p228_009.wav\"\n",
+        "audio_filepath = input_audio_dir / audio_file\n",
+        "processed_audio_filepath = output_audio_dir / audio_file\n",
+        "\n",
+        "print(\"Original audio.\")\n",
+        "ipd.display(ipd.Audio(audio_filepath))\n",
+        "\n",
+        "print(\"Processed audio.\")\n",
+        "ipd.display(ipd.Audio(processed_audio_filepath))"
+      ],
+      "metadata": {
+        "id": "_fM3GwJxkjOA"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Speaker Mapping"
+      ],
+      "metadata": {
+        "id": "d129p0nrr3PD"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "We can use [create_speaker_map.py](https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/create_speaker_map.py) to easily create a mapping from speaker ID strings to integer indices that will be used at training time.\n",
+        "\n",
+        "The script will simply sort the speaker IDs and assign them numbers [0, num_speakers) in alphabetical order."
+      ],
+      "metadata": {
+        "id": "ZJ1MWX3F3X9u"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "speaker_map_script = NEMO_SCRIPT_DIR / \"create_speaker_map.py\"\n",
+        "\n",
+        "train_manifest_filepath = DATA_DIR / \"train_manifest.json\"\n",
+        "dev_manifest_filepath = DATA_DIR / \"dev_manifest.json\"\n",
+        "speaker_filepath = DATA_DIR / \"speakers.json\"\n",
+        "\n",
+        "args = [\n",
+        "    f\"--manifest_path={train_manifest_filepath}\",\n",
+        "    f\"--manifest_path={dev_manifest_filepath}\",\n",
+        "    f\"--speaker_map_path={speaker_filepath}\"\n",
+        "]\n",
+        "\n",
+        "run_script(speaker_map_script, args)"
+      ],
+      "metadata": {
+        "id": "b5gdccYhr5Gk"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Visualize the speaker map file.\n",
+        "!head $speaker_filepath"
+      ],
+      "metadata": {
+        "id": "CMcC2Nqmt5AR"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Feature Computation"
+      ],
+      "metadata": {
+        "id": "jyFxOjy6t8vo"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Before training FastPitch, we need to compute some features for every audio file. The default [config file](https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/feature/feature_44100.yaml) we will use has parameters for computing the **pitch** and **energy** of every audio frame."
```

Comment:
nit: I saw inconsistent font formats for **pitch** and **energy**, and sometimes *pitch* and *energy*.